docker-templates/one-server.yaml

```diff
@@ -63,6 +66,7 @@ x-supervisor: &supervisor
     - ./logs:/logs/
     - ./supervisor_data:/data
     - ./config:/config/
+    - scoreboard_db:/scoreboard_db
```

Comment:
another thing that _needs_ to be documented, people are supposed to copy-paste this file iirc.